src/Symfony/Component/Config/Definition/BaseNode.php

```diff
@@ -50,6 +56,39 @@ public function __construct(?string $name, NodeInterface $parent = null, string
         $this->pathSeparator = $pathSeparator;
     }
 
+    /**
+     * Register possible (dummy) values for a dynamic placeholder value. Matching configuration values will be processed
```

Comment:
Registers

Reply:
I would move all sentences but the first one as a phpdoc description.

Reply:
You mean split into short & long description right? Did that for now.